index.html

```diff
@@ -1,19 +1,88 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+
+  </head>
+  <body>
+    <!--header-->
+    <header>
+      <a href="#"><img  class="karma__logo" src="img/karma-logo.svg" alt="Karma Logo" /></a>
+
+    <!-- Navigation Bar -->
+      <nav>
+        <ul id="navigation__list">
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Meet Karma</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">How it Works</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Store</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Blog</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Help</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+
+   <!-- Hero Section -->
+      <section id="hero">
+        <h1>Introducing Karma</h1>
+        <h2>Bring WiFi with you, everywhere you go.</h2>
+        <button>Learn More</button>
+      </section>
+
+    <!--Icons Section -->
+      <section id="icons">
+        <h1>Everyone needs a little Karma.</h1>
+        <div class="icon__images">
+          <img src="img/icon-devices.svg" alt="device image" />
```

Comment:
These images are large, have a css class to take care of the width and height property and this should be fine